jwst/datamodels/schemas/dark.schema.yaml

```diff
@@ -1,25 +1,30 @@
 title: Dark current data model
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: instrument_name.schema.yaml
+- $ref: detector.schema.yaml
+- $ref: filter_core.schema.yaml
```

Comment:
If we want to go that route, would it be beneficial to make "keyword" a prefix, as opposed to suffix, such as "keyword_filter.schema.yaml" and "keyword_exptype.schema.yaml", just so they all show up together in directory listings? There are some cases of data models where we've tried to do something like, for example having all of the various flavors of photom data model schemas start with the word "photom", so that they're easy to find and automatically get grouped together.